Lib-O5e-MonsterSupplement/o5e_ExtDB_EditActions.command

```diff
@@ -0,0 +1,60 @@
+[h: monster = arg(0)]
+
+[h: o5e_ExtDB_Constants (getMacroName())]
+[h: monsterName = json.get (monster, "name")]
+[h: finishedInput = 1]
+[h: exampleInputLabels = 
+	"junk | <html>Example actions (use <i>italicized</i> text)</html> || label | span=true" + 
```

Comment:
When labels are spanned and there's no 'value' field, the var name is tooltipped. Meaning this Label (and several others that follow) will have a tooltip of 'junk'. Maybe let's not do that.